src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/AbstractSyntaxFactsService.cs

```diff
@@ -551,6 +551,7 @@ protected int GetStartOfNodeExcludingAttributes(SyntaxNode node)
 
         public abstract SyntaxList<SyntaxNode> GetAttributeLists(SyntaxNode node);
 
+#nullable enable
```

Comment:
i'd personally prefer a whole-file approach to nullable.  If we're not ready for the full file, then don't bother.  once we are, we can just do the whole file and move on.

Reply:
I see your point but I think there's a benefit in creating islands of nullability whenever some code is touched. 

That said, I don't have strong preference on this one, just following an "ok" to move even "parts of file" by @333fred

Reply:
At least in the compiler, we are perfectly fine with enabling nullability in small sections as we work on related pieces of code.

Reply:
I took a look at these files, and i personally don't get the value of the island.  They're not uber complex areas with lots of moving parts.  We could just turn it on for the file, do the few minutes or work to make it clean, and just know we were done with it.

Reply:
TBH, not knowing the usage of most of SyntaxFacts API it's not that quick for me to decide whether we want / can have this parameter nullable or not.

I'm happy to remove the nullability island and keep just my whole files #nullable enabled if that's what's desirable.

Reply:
i'd prefer that TBH.  We can bit off syntaxfacts in one go in the future.  I'd be happy to help out there myself.  

Note: the semantics *are* supposed to be that effectively it is 'null tolerant' on all query methods (i.e. IsFoo).  But that it is null-interolerant on grabbing data off a node.  so "GetExpressionOfExpressionStatemnet" expects that you already validated what you passed in was an ExpressionStatement.

Reply:
Ok, will cleanup tomorrow.